pyfr/solvers/navstokes/elements.py

```diff
@@ -73,15 +81,24 @@ def col_view(mat):
 
                     return backend.view(matmap, rcmap, stridemap, vshape)
 
-                avis_upts_cv = col_view(avis_upts)
+                # Element-wise artificial viscosity kernel
                 avis_fpts_cv = col_view(self._avis_fpts)
                 avis_upts_temp_cv = col_view(avis_upts_temp)
 
-                # Element-wise artificial viscosity kernel
-                avis = backend.kernel('avis', tplargs, dims=[self.neles],
-                                      ubdegs=ubdegs,
-                                      s=avis_upts_temp_cv,
-                                      amu_e=avis_upts_cv, amu_f=avis_fpts_cv)
+                if 'flux' in self.antialias:
```

Comment:
Can probably clean these two blocks up a little.  In the big `backend.kernel` block, for example, the only thing that changes is `amu_e`.